File: Houseclub/src/main/java/me/grishka/houseclub/fragments/FollowingFragment.java
package me.grishka.houseclub.fragments;

import android.app.Activity;

import me.grishka.appkit.api.SimpleCallback;
import me.grishka.houseclub.R;
import me.grishka.houseclub.api.methods.GetFollowing;

public class FollowingFragment extends UserListFragment {

    @Override
    public void onAttach(Activity activity) {
        super.onAttach(activity);
        setTitle(R.string.following_title);
    }

    @Override
    protected void doLoadData(int offset, int count) {
        currentRequest = new GetFollowing(getArguments().getInt("id"), 50, offset / 50 + 1)
                .setCallback(new SimpleCallback<GetFollowing.Response>(this) {
                    @Override
                    public void onSuccess(GetFollowing.Response result) {
                        currentRequest = null;
                        onDataLoaded(result.users, data.size() + preloadedData.size() + result.users.size() < result.count);
                    }
                })
                .exec();
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/fragments/FollowingFragment.kt
package me.grishka.houseclub.fragments

import android.app.Activity
import me.grishka.appkit.api.SimpleCallback
import me.grishka.houseclub.R
import me.grishka.houseclub.api.methods.GetFollowing

class FollowingFragment : UserListFragment() {
    override fun onAttach(activity: Activity) {
        super.onAttach(activity)
        setTitle(R.string.following_title)
    }

    override fun doLoadData(offset: Int, count: Int) {
        currentRequest = GetFollowing(arguments.getInt("id"), 50, offset / 50)
            .setCallback(object : SimpleCallback<GetFollowing.Response?>(this) {
                override fun onSuccess(result: GetFollowing.Response?) {
                    currentRequest = null
                    onDataLoaded(result!!.users, data.size + preloadedData.size + result.users!!.size < result.count)
                }
            })
            .exec()
    }
}